templates/kubernetes/terraform/modules/kubernetes/files/wireguard-wg0-conf.tpl

```diff
@@ -0,0 +1,7 @@
+[Interface]
+Address = ${tpl_server_address}
+ListenPort = 51820
+PostUp = wg set wg0 private-key /etc/wireguard/privatekey && iptables -A FORWARD -s ${tpl_server_address} -d ${tpl_destination_subnets} -j ACCEPT && iptables -A FORWARD -s ${tpl_server_address} -j DROP && iptables -t nat -A POSTROUTING -s ${tpl_server_address} -o eth0 -j MASQUERADE
```

Comment:
Would it work to add \ after each line and add newlines so it's not all on one line?

Reply:
done with multiple lines